linetools/spectra/lsf.py

```diff
@@ -112,7 +112,8 @@ def check_and_reformat_relpix(self, relpix):
         """Performs checks and reformating on a relative pixel array.
 
         At the moment this function does the following:
-         - Make sure that the number of relative pixels of the LSF is odd integer
+         - Make sure that the number of relative pixels of the LSF is an odd integer
+            - Unfortunately, the STIS LSFs provided by STScI have an even number
```

Comment:
Remove this docstring comment. You have changed the STIS tables to have a odd number of rows, right?